identifier-ui/src/Auth.js

```diff
@@ -14,7 +14,7 @@ const Auth = ({onSuccess, onFailure}) => {
     };
     var authenticationDetails = new cognito.AuthenticationDetails(authenticationData);
     var userData = {
-      Username : 'EthanTheMan',
+      Username : authenticationData.Username,
```

Comment:
ah I didn't see this other usage - we can just use the state variable `username` for simplicity
```suggestion
      Username : username,
```